src/status_im/contexts/wallet/add_address_to_watch/view.cljs

```diff
@@ -11,19 +11,16 @@
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
-(defn validate-message
-  [addresses]
-  (fn [s]
-    (cond
-      (or (= s nil) (= s ""))             nil
-      (contains? addresses s)             (i18n/label :t/address-already-in-use)
-      (not (or (validation/eth-address? s)
-               (validation/ens-name? s))) (i18n/label :t/invalid-address)
-      :else                               nil)))
+(defn- validate-address
+  [known-addresses s]
```

Comment:
can we have a more descriptive variable name for `s`?

Reply:
sure

Reply:
done